### PR TITLE
Add translation mechanism for "Objective"

### LINKED ISSUE #321: Add translation mechanism for "Objective"
String "Objective" can't be translated to any language except English.

image

### PR DESCRIPTION
**Pull request by alexandear**
Resolves #321

### CODE REVIEW

File: src/js/intl/strings.js [hunk 1]
```diff
@@ -917,6 +917,12 @@ exports.strings = {
     'ru_RU': 'Спрятать цель'
   },
   ///////////////////////////////////////////////////////////////////////////
+  'objective-button': {
```
Comment (pcottle) [inline src/js/intl/strings.js]: yep youre right -- i guess i had never wrapped this in an intl call! I thought maybe the token was in this file and I just missed it in the transition

File: src/js/react_views/LevelToolbarView.jsx [hunk 2]
```diff
@@ -70,7 +70,7 @@ var LevelToolbarView = React.createClass({
             <button
               onClick={this.props.onObjectiveClick}
               type="button">
-              Objective
+              {intl.str('objective-button')}
```
Comment (pcottle) [inline src/js/react_views/LevelToolbarView.jsx]: :+1:

**Comment by pcottle** [comment]
Thanks so much @alexandear, officially a contributor to the project :tada: just pushed the site with your updates:

```
To git@github.com:pcottle/learnGitBranching.git
   db2cff1..30c8a26  gh-pages -> gh-pages
Switched to branch 'master'
Your branch is up-to-date with 'origin/master'.
Done, without errors.
```

**Comment by alexandear** [comment]
thanks

**Pull request merged**